app/ui.js

```diff
@@ -502,7 +502,6 @@ var UI;
             }
 
             UI.enableDisableViewClip();
-            document.getElementById('noVNC_setting_resize').disabled = UI.connected;
```

Comment:
Isn't more code required? Don't we have to trigger an update of the canvas size and position?